dis_snek/client/utils/cache.py

```diff
@@ -1,48 +1,53 @@
 import time
 from collections import OrderedDict
 from collections.abc import ItemsView, ValuesView
-from typing import Any, Iterator, Tuple
+from typing import Any, Callable, Generic, Iterator, Optional, Tuple, TypeVar
 
 import attrs
 
 from dis_snek.client.utils.attr_utils import define, field
 
 __all__ = ["TTLItem", "TTLCache"]
 
+KT = TypeVar("KT")
+VT = TypeVar("VT")
```

Comment:
Better to put all the generic `TypeVar` variables in const.py? Since its likely being used in multiple files

Reply:
If I end up having to type more mappings, I'll move them